theme/basics/template-hierarchy.md

```diff
@@ -424,79 +189,35 @@ Rendering an attachment page (`attachment` post-type) uses the following path:
 5.  `single.php`
 6.  `singular.php`
 7.  `index.php`
--->
-1. `{MIME-type}.php` – 任意の[MIME タイプ（メディアタイプ）](https://ja.wikipedia.org/wiki/%E3%83%A1%E3%83%87%E3%82%A3%E3%82%A2%E3%82%BF%E3%82%A4%E3%83%97) (例: `image.php`, `video.php`, `pdf.php`)。`text/plain` ならば以下の順で表示されます。
-    1.  `text-plain.php`
-    2.  `plain.php`
-    3.  `text.php`
-2. `attachment.php`
-3. `single-attachment-{slug}.php` – 例えば、添付ファイルのスラッグが `holiday` であれば、WordPress は `single-attachment-holiday.php` を探します。
-4. `single-attachment.php`
-5. `single.php`
-6. `singular.php`
-7. `index.php`
-
-<!-- 
+
 ### Embeds
--->
-### 埋め込み
 
-<!-- 
 The embed template file is used to render a post which is being embedded. Since 4.5, WordPress uses the following path:
--->
-埋め込みテンプレートファイルは、埋め込み対象の記事を表示するために使用されます。4.5 以降、WordPress は以下の順で表示します。
 
-<!-- 
 1.  `embed-{post-type}-{post_format}.php` – First, WordPress looks for a template for the specific post. For example, if its post type is `post` and it has the audio format, WordPress would look for `embed-post-audio.php`.
 2.  `embed-{post-type}.php` – If the post type is `product`, WordPress would look for `embed-product.php`.
 3.  `embed.php` – WordPress then falls back to embed`.php`.
 4.  Finally, WordPress ultimately falls back to its own `wp-includes/theme-compat/embed.php` template.
--->
-1. `embed-{post-type}-{post_format}.php` – まず、WordPressは特定の投稿のためのテンプレートを探します。例えば、投稿タイプが `post` で、オーディオ形式であれば、WordPress は `embed-post-audio.php` を探します。
-2. `embed-{post-type}.php` – 投稿タイプが `product` ならば WordPress は `embed-product.php` を探します。
-3. `embed.php` – 次に WordPress は `embed.php` をフォールバックとします。
-4. 最終的に、WordPress は自身のテンプレートである `wp-includes/theme-compat/embed.php` をフォールバックとします。
 
-<!-- 
 ## Non-ASCII Character Handling
--->
-## 非 ASCII 文字の取り扱い
 
-<!-- 
 Since WordPress 4.7, any dynamic part of a template name which includes non-ASCII characters in its name actually supports both the un-encoded and the encoded form, in that order. You can choose which to use.
```

Comment:
<!-- 
Since WordPress 4.7, any dynamic part of a template name which includes non-ASCII characters in its name actually supports both the un-encoded and the encoded form, in that order. You can choose which to use.
-->
WordPress 4.7以降、テンプレート名の動的部分に非 ASCII 文字が含まれている場合、実際にはエンコードされていない形式とエンコードされた形式の両方を順にサポートしています。どちらを使うか選択することができます。